cmd/fleet/serve.go

```diff
@@ -728,7 +728,6 @@ func cronVulnerabilities(
 		select {
 		case <-ticker.C:
 			level.Debug(logger).Log("waiting", "done")
-			ticker.Reset(config.Vulnerabilities.Periodicity)
```

Comment:
@michalnicp, my understanding of intent for this reset was to allow the scan to run the first time after only 10 seconds  and thereafter to run at the specified interval. I could easily be mistaken though.

Reply:
Good catch, I think I removed this during development due to an issue on my end with configuring the periodicity.